data/ecosystems/d/dipdup.toml

```diff
@@ -95,26 +95,3 @@ url = "https://github.com/dipdup-io/tzprofiles"
 
 [[repo]]
 url = "https://github.com/dipdup-io/workerpool"
-
-[[repo]]
-url = "https://github.com/dipdup-net/aiosignalrcore"
-
-[[repo]]
-url = "https://github.com/dipdup-net/hicdex"
-
-[[repo]]
-url = "https://github.com/dipdup-net/openapi-typescript-codegen"
-
-[[repo]]
-url = "https://github.com/dipdup-net/reorg-tracker"
-
-[[repo]]
-url = "https://github.com/dipdup-net/tezos-profiles"
-missing = true
-
-[[repo]]
-url = "https://github.com/dipdup-net/tzprofiles"
-
-[[repo]]
-url = "https://github.com/dipdup-net/wiki.tezos.com"
-missing = true
```

Comment:
All looks good, but please leave these repos there.  The org will automatically populate new repos in our infra, but we leave the full repo urls in the taxonomy too.

Reply:
Thank you for your review! Returned it back